python/ray/tests/test_task_events.py

```diff
@@ -198,37 +198,38 @@ def task():
     )
 
 
-def test_failed_task_removed_placement_group(shutdown_only, monkeypatch):
-    ray.init(num_cpus=2, _system_config=_SYSTEM_CONFIG)
-    from ray.util.placement_group import placement_group, remove_placement_group
-    from ray.util.scheduling_strategies import PlacementGroupSchedulingStrategy
-
-    pg = placement_group([{"CPU": 2}])
-    ray.get(pg.ready())
-
-    @ray.remote(num_cpus=2)
-    def sleep():
-        time.sleep(999)
-
-    with monkeypatch.context() as m:
-        m.setenv(
-            "RAY_testing_asio_delay_us",
-            "NodeManagerService.grpc_server.RequestWorkerLease=3000000:3000000",
-        )
-
-        sleep.options(
-            scheduling_strategy=PlacementGroupSchedulingStrategy(placement_group=pg),
-            name="task-pg-removed",
-            max_retries=0,
-        ).remote()
-
-    remove_placement_group(pg)
-
-    wait_for_condition(
-        verify_failed_task,
-        name="task-pg-removed",
-        error_type="TASK_PLACEMENT_GROUP_REMOVED",
-    )
+# TODO(rickyx): Make this work.
```

Comment:
So the test wasn't actually run in CI...because of the missing `__name__` thing added in this PR. 

And I believe this test wasn't working because it's hard to reproduce a placement group removed case...